TISServices/TISServices/Authorization/AuthorizeOpenAMAttribute.cs

```diff
@@ -56,7 +56,7 @@ public override void OnAuthorization(HttpActionContext actionContext)
              {
                  using (HttpClient client = new HttpClient())
                  {
-                     response = client.GetAsync(String.Format("{0}?access_token={1}", OpenAMUrl, authKey)).Result;
+                     response = client.GetAsync(String.Format("{0}?access_token={1}&realm=/sbac", OpenAMUrl, authKey)).Result;

```

Comment:
Was it not working for you without the realm?  I'm asking because we have stood TIS up and used it and we did not need to change this.